### ISSUE TITLE
Work on the exercises.

### ISSUE THREAD

**Issue by s-matyukevich** [s-matyukevich/raspberry-pi-os#6]
You can submit a PR as soon as an exercise is ready. Other people then will be able to use your code for reference.

**Comment by evopen** [comment]
If every one submit their exercises to the repo, this repo might get bloated. After all most of the code are pretty similar.

**Comment by s-matyukevich** [comment]
I don't think this is a problem right now. For now the main goal is to get at least one reference implementation for all exercises. Later, however, I may stop receiving PRs for the exercises that got too many PRs already, unless a PR is indeed a unique one.
